### ISSUE TITLE
VS Code keeps loading untitled documents on start even users closed them and chose not to save them

### ISSUE THREAD

**Issue by jeremymeng** [microsoft/vscode#91220]
- VSCode Version: 1.42.1
- OS Version: Windows 10

Steps to Reproduce:

1. our extension opens two `untitled:` documents then set the content of the documents one more more times to show decompiled code for a .NET assembly.

https://github.com/icsharpcode/ilspy-vscode/blob/7d808d7894fe1ea671bcd1cfd28c47ec5814146a/vscode-extension/src/extension.ts#L126-L136

The problem is that even after users close those document and choose not to save the files, the next time VS Code starts it loads them again, with empty content.  And [it is very hard for users to figure out how to prevent VS Code to open the untitled documents](https://github.com/icsharpcode/ilspy-vscode/issues/47#issuecomment-589509000). 

Expected:

VS Code should not try to load the file when users explicitly close and chose not to save them.

Does this issue occur when all extensions are disabled?: N/A this repros using the extension.

Basically I want to use two temporary documents to show C# and IL code and I want them gone when VS Code exits and not loaded automatically by VS Code next time.  Please let me know if there are better ways.  I don't like having to create real files under %TEMP% directory.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by jeremymeng** [comment]
The issue also repros in VS Code Insider

![image](https://user-images.githubusercontent.com/7583839/78419945-91b6ad80-75ff-11ea-801c-41148f786f2a.png)

**Comment by jeremymeng** [comment]
Is there no way to re-activate this issue? I don't see any button other than "Comment" one
